### PR TITLE
Integrated changes to blueprints for hortonworks-on-centos template

### PR DESCRIPTION
**Pull request by predeekc**

### CODE REVIEW

File: hortonworks-on-centos/azuredeploy.json [hunk 1]
```diff
@@ -126,7 +126,7 @@
             "nameNodeCount": 1,
             "dataNodeCount": 3,
             "masterVmSize": "Standard_A3",            
-            "masterNodeTemplateUri": "/cluster-node-16disk.json",
+            "masterNodeTemplateUri": "/cluster-node-8disk.json",
             "workerVmSize": "Standard_A3",
```
Comment (johngossman) [inline hortonworks-on-centos/azuredeploy.json]: Do we want to use D sizes?

File: hortonworks-on-centos/scripts/mountDisk.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DRIVE=$1
+INDEX=$2
+
+echo "Creating filesystem on $DRIVE..."
+mke2fs -F -t ext4 -b 4096 -O sparse_super,dir_index,extent,has_journal -m1 $DRIVE
+
+echo "Mountint disk $DRIVE at /disks/$INDEX"
```
Comment (johngossman) [inline hortonworks-on-centos/scripts/mountDisk.sh]: Mounting?

File: hortonworks-on-centos/azuredeploy.json (hunk 1 above)
Comment (predeekc) [inline hortonworks-on-centos/azuredeploy.json]: Not sure what you mean by D, but this change is correct.  It should have been using only 8 disks with the A3 VM

File: hortonworks-on-centos/scripts/mountDisk.sh (hunk 2 above)
Comment (predeekc) [inline hortonworks-on-centos/scripts/mountDisk.sh]: Fixing this now, good catch

**Comment by predeekc** [comment]
The spelling error is fixed

**Pull request merged**